jmclient/jmclient/taker_utils.py

```diff
@@ -195,7 +195,7 @@ def unconf_update(taker, schedulefile, tumble_log, addtolog=False):
     #full record should always be accurate; but TUMBLE.log should be
     #used for checking what actually happened.
     completion_flag = 1 if not addtolog else taker.txid
-    taker.schedule[taker.schedule_index][5] = completion_flag
+    taker.schedule[taker.schedule_index][-1] = completion_flag
```

Comment:
This is a good change, but it reminds me of a minor point: changing the schedule format could conceivably cause a problem for someone running old code then switching to a new update including this change, and having a failure on the basis of the change in `.schedule` format. It's unlikely but we must at least note it carefully in release notes. Also have a quick think, could it conceivably cause any more serious problem? I think not.

Reply:
I agree, I've opened an issue #429 about the problems with the CSV file format used for schedule.